index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+<!-- The Links --> 
+<link rel="stylesheet" href="style.css">
+<link rel="stylesheet" href="font-awesome/css/font-awesome.css"> 
+</head>
+<body>
+<!-- The Header -->   
+  <header class="header"> 
+  <div class="contect-info" >
+    <span class="time" >12:00</span>  
+    <span class="fa fa-battery-full"></span> 
+    <span class="fa fa-wifi"></span>
+    <span class="fa fa-signal"></span>
+  </div>
+  <div >  
+      <span class="fa fa-bars" id="open"></span>  
+      <h1 class="logo">Gulalai Company</h1> 
+         
+  </div>
+
+  </header>
+  <section class="explain container">
+    <h1>Hey There.
+      I’m Gulalai</h1> 
+    <h3>I’am a Software Developer</h3>
+    <h5> I can help you build a product , 
+      feature or website Look through some of my work and experience! 
+      If you like what you see and have a project you need coded, 
+      don’t hestiate to contact me.</h5> 
+      <div class="contact ">
+        <ul class="container">
+          
+          <li><a href="#"><i class="fa fa-linkedin" ></a></i></li>
+          <li><a href="#"><i class="fa fa-twitter" ></a></i></li>
+          <li><a href="#"><i class="fa fa-github" ></a></i></li>
+          <li><a href="#"><i class="fa fa-angellist" ></a></i></li>
+          <li><a href="#"><i class="fa fa-medium" ></a></i></li> 
+      </ul>
```

Comment:
- [ ] The list of social icons are not appearing in my display. Please make sure you imported it correctly, the HTML tags are placed correctly and they are displayed. 
 -Your design:
<img width="395" alt="image" src="https://user-images.githubusercontent.com/61919617/192575187-1b59d35f-4918-472d-b10e-449bcd1d2316.png">